src/public/modules/forms/services/submissions.client.factory.js

```diff
@@ -46,8 +46,17 @@ function SubmissionsFactory(
   const publicSubmitUrl = '/v2/submissions/:responseMode/:formId'
   const previewSubmitUrl = '/v2/submissions/:responseMode/preview/:formId'
 
+  const API_VERSION_PREFIX = '/api/v3'
+  const ADMIN_FORMS_PREFIX = '/admin/forms'
```

Comment:
nit: don't need to split up the constants, just call it `ADMIN_FORMS_BASE` or something and have the full URL there since there are no other versions

Reply:
will combine both into `ADMIN_FORMS_PREFIX`